README.md

```diff
@@ -0,0 +1,105 @@
+#dp-logging
+
+Uses logback and provides extendable log message builder for structured logging.
+
+## Example
+Extend the `LogMessageBuilder`:
+```java
+public class LogBuilderExample extends LogMessageBuilder {
+
+    /**
+     * static convenience method for info level logging.
+     */
+    public static LogBuilderExample info(String message) {
+        return new LogBuilderExample(message);
+    }
+
+    private LogBuilderExample(String eventDescription) {
+        super(eventDescription);
+    }
+
+    private LogBuilderExample(String description, Level logLevel) {
```

Comment:
Consider removing the constructor overloads specifying log levels if we are not using them.

Reply:
This is a read me - agree it could be simplified but I want to minimise the impact of the change in Zebedee - as it stands this change does not break the interface it just changes the internal logic.